src/tests/VisualStudioProjectLoaderTests.cs

```diff
@@ -142,6 +144,38 @@ public void CanLoadVsProject(string resourceName, string[] configs, string assem
             }
         }
 
+        [TestCase("netcoreapp1.1-minimal.csproj", "Debug", @"bin\Debug\netcoreapp1.1")]
+        [TestCase("netcoreapp1.1-minimal.csproj", "Release", @"bin\Release\netcoreapp1.1")]
+        [TestCase("netcoreapp1.1-with-output-path.csproj", "Debug", @"bin\Debug\netcoreapp1.1")]
+        [TestCase("netcoreapp1.1-with-output-path.csproj", "Release", @"bin\Release\special")]
```

Comment:
I'll change to forward slashes. I don't have a Linux environment available to confirm, though.